analysis/pyenv.sh

```diff
@@ -30,21 +30,32 @@ ENV=.pyenv
 DATAGUI=../data-gui/
 export VIRTUAL_ENV_DISABLE_PROMPT=1
 
+PYTHON=python3
+PIP=pip3
+command -v $PYTHON > /dev/null 2>&1
+if [ $? -ne 0 ]; then
```

Comment:
Could you pls switch to bash syntax with double `[[` `]]`, just for consistency?